terraform/environments/corporate-staff-rostering/locals_production.tf

```diff
@@ -243,6 +243,34 @@ locals {
         }
       }
 
+      pd-csr-a-10-b = {
+        config = merge(local._defaults_app_ec2.config, {
+          ami_name          = "pd-csr-a-10-b"
+          availability_zone = "${local.region}b"
+        })
+        instance = merge(local._defaults_app_ec2.instance, {
+          instance_type = "m5.4xlarge"
+        })
+        ebs_volumes = {
+          "/dev/sda1" = { type = "gp3", size = 128 } # root volume
+          "/dev/sdb"  = { type = "gp3", size = 128 }
+          "/dev/sdc"  = { type = "gp3", size = 112 }
+          "/dev/sdd"  = { type = "gp3", size = 128 }
```

Comment:
Please check these volumes against the AMI. The sizes don't match with the device names.